src/primer_bed.cpp

```diff
@@ -127,9 +128,9 @@ std::vector<primer> populate_from_file(std::string path){
 	if(std::all_of(cell.begin(), cell.end(), ::isdigit)) {
 	  p.set_score(stoi(cell));
 	} else {
-	  print_bed_format();
-	  primers.clear();
-	  return primers;
+	  print_bed_format();  // score is missing, send warning but continue populating
```

Comment:
Warning seems too wordy. The following should suffice.
`WARNING: The provided BED file did not have the expected score column but iVar will continue trimming.`